src/components/courses/AdminPage.tsx

```
@@ -246,29 +291,79 @@ const AdminPage = () => {
+                                content: exam.content, // Sửa thành exam.content
+                                isComplete: exam.isComplete, // Dữ liệu từ exam
+                                filesExam: exam.filesExam, // Sử dụng đúng fileExams từ exam
+                                fileSubmission: exam.fileSubmission, // Sử dụng đúng fileSubmission từ exam
+                              });
+                              setIsEdit(true);
+                            }}
+                            className="text-blue-500 hover:text-blue-700"
+                          >
+                            <FaEdit />
+                          </button>
+                          <button
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)